dist/windows/CMakeLists.txt

```diff
@@ -1 +1,13 @@
-install(FILES qt.conf DESTINATION ${CMAKE_INSTALL_BINDIR})
+add_custom_target(copy_qt_conf
```

Comment:
Why not `file(COPY`?

Reply:
@zeule Because it is my impression that it cannot use generator expressions? But please correct me if I'm wrong.

Reply:
Ah, yes, you are right... But why do you need `$<CONFIG>` there?

Reply:
@zeule because of:

```cmake
set_target_properties(qBittorrent
    PROPERTIES
        RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/$<CONFIG>"
)
```

in `src/app` CML. It is super convenient for single- and multi-config generators alike. Binaries produced from different configs end up in separate folders for each config. `qt.conf` is supposed to be in the same directory as the executable.

Reply:
This one makes it unusable. It cannot be run from Qt Creator since it is unable to find installed Qt components.
Why do you really need to copy it there? It's needed to in install directory only.

Reply:
>Binaries produced by different configs end up in separate folders for each config. 

The convenience is questionable, since we are only talking about the resulting binary. Intermediate files are still created in the same directory, so if I need to support multiple configurations at the same time, I still have to set the different build directory per configuration manually, and this only adds an unnecessary subfolder inside, for example "qBittorrent-build/debug/Debug".

Reply:
https://github.com/qbittorrent/qBittorrent/pull/12746/files#discussion_r449767515 is blocking issue.

Reply:
@glassez 

> The convenience is questionable, since we are only talking about the resulting binary. Intermediate files are still created in the same directory, so if I need to support multiple configurations at the same time, I still have to set the different build directory per configuration manually, and this only adds an unnecessary subfolder inside, for example "qBittorrent-build/debug/Debug".

You're right. Next fixup commit will change this back (so that the executable ends up under `src/app/` like before).

> Why do you really need to copy it there? It's needed to in install directory only.

The point of copying `qt.conf` to the same location as the executable in the build tree is so that those options are still applied when quickly building and testing without installing.

Reply:
>The point of copying qt.conf to the same location as the executable in the build tree is so that those options are still applied when quickly building and testing running the executable from the build tree without installing.

But really it breaks "running the executable from the build tree"!
What do you think the purpose of this file is?

Reply:
If you didn't understand, placing qt.conf near the qBittorrent executable in output directory is problem.

Reply:
> You're wrong! Currently (before your patch) it is placed in root of build tree (this was changed lately from the place you mentioned above).

Reply:
Right, I understand now. Will remove the the build-time copying of this file.